src/Directory.Build.targets

```diff
@@ -6,4 +6,15 @@
     <Exclude>[GraphQL.StarWars]*,[GraphQL.Harness]*,[GraphQL.Dummy]*</Exclude>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(IsPackable)' == 'true'">
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <WarningsAsErrors>$(WarningsAsErrors);CS1591;CS1574;CS8765;CS8601;CS8620;CS8600;CS8625;CS8767;CS8603;CS8618</WarningsAsErrors>
```

Comment:
I suggest to switch to `<TreatWarningsAsErrors>true</TreatWarningsAsErrors> + <WarningsNotAsErrors>X;Y;Z</WarningsNotAsErrors>` instead.

Reply:
Done